hypervisor/virt/ahv/ahvncli.py

```diff
@@ -0,0 +1,63 @@
+from hypervisor import FailException
+from hypervisor import logger
+from hypervisor.ssh import SSHConnect
+
+
+class AHVnCLI:
+    def __init__(self, server, ssh_user, ssh_pwd):
+        """
+        Collect AHV information, provide Guest add/delete/start/stop/suspend/resume functions
+        via ncli Command line following 'https://portal.nutanix.com/page/documents/details?
+        targetId=Command-Ref-AOS-v6_0:man-ncli-c.html'
+        :param server: the ip for the AHV server
+        :param ssh_user: the ssh user for the AHV server
+        :param ssh_pwd: the ssh password for the AHV server
+        """
+        self.server = server
+        self.ssh_user = ssh_user
+        self.ssh_pwd = ssh_pwd
+        self.ssh = SSHConnect(self.server, user=self.ssh_user, pwd=self.ssh_pwd)
+
+    def get_ncli_info(self, cmd, value):
+        """
+        Get the info from AHV
+        :param cmd:
+        :param value: the value you want to get from the result
```

Comment:
I think we should give an example for the both cmd and value,

Reply:
will update this function with another suitable way :)